### PR TITLE
Add labeler workflow

### PR DESCRIPTION
**Pull request by kingthorin**
I am HORRIBLE at remembering to add the default label. I know this isn't 100% applicable but it is more often than not. So for the few times it needs to be removed to be replaced by enhancement or something I think that's fine.

Ref: https://github.com/actions/labeler

### CODE REVIEW

**Comment by psiinon** [comment]
Me too ;) So what does this actually do?

**Comment by kingthorin** [comment]
So any time a PR is opened it'll apply the `Type-Task` label automatically.

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: "Pull Request Labeler"
```
Comment (thc202) [inline .github/workflows/labeler.yml:1]: Maybe change the name of the file too? I'd like to have a e.g. `to-triage` label for issues.

Comment (kingthorin) [reply .github/workflows/labeler.yml:1]: Happy to rename things.

I'm not going to add the other label/confit/workflow until we discuss how that process will work and how/when we're considering triage complete. (That's probably for another PR.)

Comment (kingthorin) [reply .github/workflows/labeler.yml]: Tweaked.

Comment (thc202) [reply .github/workflows/labeler.yml]: Yeah, not asking for a new workflow here, just giving the reason to use a more specific name.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by ricekot — APPROVED** [review]
Thanks!

**Pull request merged**